YogaKit/YGLayout+Private.h

```diff
@@ -8,8 +8,11 @@
  */
 
 #import "YGLayout.h"
+#import <yoga/Yoga.h>
 
-@interface YGLayout (Private)
+@interface YGLayout ()
```

Comment:
That's required because the private header now contains a property (`node`, which needs to be accessed in the tests) and which isn't synthesised if declared in a category, compared to a private interface definition.